CppCoreGuidelines.md

```diff
@@ -927,7 +927,7 @@ Controlling the behavior of a function through a global (namespace scope) variab
 It will not be obvious to a caller that the meaning of two calls of `rnd(7.2)` might give different results.
 
 **Exception**: Sometimes we control the details of a set of operations by an environment variable, e.g., normal vs. verbose output or debug vs. optimized.
-The use of a non-local control is potentially confusing, but controls only implementation details of an otherwise fixed semantics.
+The use of a non-local control is potentially confusing, but controls only implementation details of an otherwise fixed semantic.
```

Comment:
Yeah it could be, but how it stands now it reads 'an semantics' which is incorrect. Could be fixed either way I think ('a set of semantics' or 'a semantic') just depends on the intent